src/app/components/send-coins-form/send-coins-form.component.ts

```diff
@@ -80,10 +109,21 @@ export class SendCoinsFormComponent implements OnInit {
     this.originCoin = data.originCoin ? data.originCoin : undefined
     this.destCoin = data.destCoin
     this.destAddr = data.destAddr
+
+    if( data.errors.length === 0 ) {
+      this.estimateValid = true
+      setTimeout(() => {
```

Comment:
we should potentially store this so we can cancel it? Not sure if its necessary, but perhaps we might have issues with the timer triggering later or duplicating if we go back and forth between states.